crates/vizia_core/src/views/slider.rs

```diff
@@ -97,14 +99,15 @@ where
     /// ```
     pub fn new(cx: &mut Context, lens: L) -> Handle<Self> {
         Self {
-            p: PhantomData::default(),
+            lens: lens.clone(),
```

Comment:
Since `PhantomData` isn't being used any more can you remove the import to remove the warning.